Source/WebKit/Shared/UserData.cpp

```diff
@@ -87,7 +87,8 @@ static bool shouldTransform(const API::Object& object, const UserData::Transform
             if (!keyValuePair.value)
                 continue;
 
-            if (shouldTransform(*keyValuePair.value, transformer))
+            Ref value = *keyValuePair.value;
+            if (shouldTransform(value, transformer))
```

Comment:
Just curious — any reason why we don't use `shouldTransform(Ref { *keyValuePair.value }, transformer)` instead?

Reply:
I was just trying out a different style but will use that instead.